src/components/graphs/PieChart.vue

```diff
@@ -1,12 +1,21 @@
 <template>
   <div class="pie-chart-cont">
     <svg id="pie-chart"><!-- D3 inserts here --></svg>
+    <img
+      v-tooltip.bottom="tooltipMessage"
+      class="tooltip"
+      src="/help.svg"
+      alt="Help icon. Hover to reveal additional text."
```

Comment:
Alt text is typically exposed to screen reader users, who don't use a mouse. I'd just change this to "Help Icon"